src/components/App/App.js

```diff
@@ -39,11 +41,17 @@ class App extends Component {
         path="/movies/:id/" exact
         render={({ match }) => {
           const selectedMovie = this.props.movies.find(movie => parseInt(match.params.id) === movie.id)
+
           return <MovieDetails
                   {...selectedMovie}
                  />
         }}
       />
+      <Route
+        path="*"
+        component={MovieContainer}
+      />
+      </Switch>
       </main>
```

Comment:
Good idea to add the switch function. This will help us redirect users to the main page if they enter an invalid url.